omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml

```diff
@@ -345,7 +345,14 @@
                     if (viral_load_status == 1301) {
                         /* Detected */
                         jq("#viral-load-results-copies").attr("disabled", false).attr("readonly", false).fadeTo(250, 1);
-                        setValue('viral-load-results-copies.value', '');
+                        <ifMode mode="ENTER">
+                                setValue('viral-load-results-copies.value', '');
+                        </ifMode>
+
+                        var viral_load_copies = getValue('viral-load-results-copies.value');
```

Comment:
@dbaluku No need to clear values on edit so this additional check is not needed

Reply:
@ssmusoke the addition check is needed because it clears the value of copies if the VL_qualitative is detected and VL_copies is equal 1.
This is for cases when on edit someone chooses another qualitative value like UNDETECTED